core/src/test/java/org/elasticsearch/search/aggregations/bucket/histogram/InternalDateHistogramTests.java

```diff
@@ -55,7 +55,7 @@ protected InternalDateHistogram createTestInstance(String name,
                                                        List<PipelineAggregator> pipelineAggregators,
                                                        Map<String, Object> metaData,
                                                        InternalAggregations aggregations) {
-        int nbBuckets = randomInt(10);
+        int nbBuckets = randomNumberOfBuckets();
```

Comment:
I think 10 would be fine as a default as well instead of 25, it should be enough for meaninful tests.